lib/ansible/modules/system/lvg.py

```diff
@@ -44,6 +44,12 @@
     - Additional options to pass to C(pvcreate) when creating the volume group.
     type: str
     version_added: "2.4"
+  pvresize:
+    description:
+    - If C(yes) resize the physical volume to the maximum available size
+    type: bool
+    default: yes
```

Comment:
The goal is to keep compatibility, and default to "no" would be safe in this case, (except it make sense to change the behaviour). Could you explain why it would be good to change the default to "yes"? 

If yes is the default, I would suggest to make an entry in the porting guide https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/porting_guides/porting_guide_2.10.rst.

Reply:
changed default behaviour to false